# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/webrtc/jsapi/PeerConnectionImpl.h
# Commit: a55a718715f6
# Full Hash: a55a718715f69cba1ff9fbae4787881b4d930907
# Author: Andreas Pehrson <apehrson@mozilla.com>
# Date: 2021-11-01 18:17:33
# Regressor Bug: 1654112
# File Overlap Count: 3
# Description:
#   Bug 1654112 - Report video frame-/bitrate telemetry once per call instead of every second. r=ng
#   
#   This is similar to the behavior prior to bug 1532898, which introduced telemetry
#   reporting every second. There's a bit of a difference in that we now only report
#   data if we had collected some through the 1 second interval telemetry timer in
# ==============================================================================

diff -r 069c5baf6d32 -r a55a718715f6 dom/media/webrtc/jsapi/PeerConnectionImpl.h
--- a/dom/media/webrtc/jsapi/PeerConnectionImpl.h	Thu Jan 28 10:54:32 2021 +0100
+++ b/dom/media/webrtc/jsapi/PeerConnectionImpl.h	Mon Jan 25 16:54:14 2021 +0100
@@ -412,7 +412,7 @@
   RefPtr<dom::RTCStatsReportPromise> GetStats(dom::MediaStreamTrack* aSelector,
                                               bool aInternalStats);
 
-  void RecordConduitTelemetry();
+  void CollectConduitTelemetryData();
 
   // for monitoring changes in track ownership
   // PeerConnectionMedia can't do it because it doesn't know about principals